docs/index.rst

```diff
@@ -33,10 +33,12 @@ Extensions
 * `sphinx-notfound-page`_: renders a nice 404 page respecting all the look & feel of your documentation
 * `sphinx-version-warning`_: adds a warning banner at the top if reader is reading an old version of your documentation
 * `sphinx-hoverxref`_: adds tooltips on cross references of the documentation with the content of the linked section
+* `sphinx-last-updated-by-git`_: adds the "last updated" date for each page (obtained from the Git commit date)
```

Comment:
Does it worth to add a specific page for this extension with an example as we are doing with the other extensions? For example, https://sphinx-extensions.readthedocs.io/en/latest/sphinxemoji.html

(the ones that I'm just linking to the docs here is because I had not time to wrote them :sweat: )

Reply:
I don't know. There is nothing to show (since the date appears on all pages anyway) and installation and usage are trivial.

But if you think every extension should have its own page, we can add a bare-bones page.

But it would probably be better to fix the `toctree` structure first (see #8)?

Reply:
OK! I will merge this as it is. We can improve it later.

(I'm not having too much time :cry: for all the things I'm doing and I don't want to block this forever)